### PR TITLE
Fix modular-scripts builds are failing because of postcss upgrade

### PR DESCRIPTION
**Pull request by cristiano-belloni**
- Bump up postcss-loader and use `implementation` to use postcss 8
- Change the webpack conf to use the new conf schema
- ~Don't lint /config. Not really sure about this, but it seems that modular lint will always die if webpack.config.js is modified and will not let the user commit (because it's invoked by lint-staged), because it tries to run jest on a pattern it doesn't match - what do you think @LukeSheard?~ - reverted, another ticket for this

### CODE REVIEW

File: packages/modular-scripts/src/lint.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ async function lint(
     displayName: 'lint',
     rootDir: modularRoot,
     testMatch: targetedFiles,
-    testPathIgnorePatterns: ['/node_modules/', '/dist/'],
+    testPathIgnorePatterns: ['/node_modules/', '/dist/', '/config/'],
```
Comment (LukeSheard) [inline packages/modular-scripts/src/lint.ts:53]: Let's leave this for now and fix in a different commit by changing the file structure?

Comment (cristiano-belloni) [reply packages/modular-scripts/src/lint.ts:53]: If I remove it in the next commit I think it will work - wdym by "changing the file structure"?

Comment (cristiano-belloni) [reply packages/modular-scripts/src/lint.ts:53]: ah, ok, you want to move it under src, ok

Comment (cristiano-belloni) [reply packages/modular-scripts/src/lint.ts]: 👍🏻 Removed, we'll capture this in another ticket

**Review by LukeSheard — APPROVED** [review]

**Pull request merged**